qemu/tests/block_hotplug.py

```diff
@@ -64,6 +65,22 @@ def unplug_device(vm, get_disk_cmd, device):
         """
         disks_before_unplug = find_disk(vm, get_disk_cmd)
         device.unplug(vm.monitor)
+
+        event_acquired = False
+        deadline = time.time() + float(params.get('event_timeout'))
+        while time.time() < deadline:
+            if vm.monitor.get_event(params.get('event_name')):
+                logging.debug("Get event \"%s\" info after unplug device: \n%s"
+                              % (params.get('event_name'),
+                                 vm.monitor.get_event(params.get('event_name'))))
+                vm.monitor.clear_event(params.get('event_name'))
+                event_acquired = True
+                break
+            time.sleep(float(params.get('check_interval_timeout')))
+        if not event_acquired:
+            test.fail("Can not get event \"%s\" under %s."
+                      % (params.get('event_name'), params.get('event_timeout')))
+
```

Comment:
Hi yhong, these codes looks as a part of verify device unplugged, if add them to verify_unplug() is better?
and for title word "fuction" miss letter "n".

Reply:
I don't think so, since here just get a deleted event info, not a action for verifying.